src/Model/ModelData.php

```diff
@@ -326,14 +333,10 @@ public function setCustomisedObj(ModelData $object)
     // CASTING ---------------------------------------------------------------------------------------------------------
 
     /**
-     * Return the "casting helper" (a piece of PHP code that when evaluated creates a casted value object)
+     * Return the "casting helper" (an injectable service name)
      * for a field on this object. This helper will be a subclass of DBField.
-     *
-     * @param bool $useFallback If true, fall back on the default casting helper if there isn't an explicit one.
-     * @return string|null Casting helper As a constructor pattern, and may include arguments.
-     * @throws Exception
      */
-    public function castingHelper(string $field, bool $useFallback = true): ?string
+    public function castingHelper(string $field): ?string
```

Comment:
The `$useFallback` was an early addition to CMS 6 - so we don't even need to deprecate it! 🎉

Reply:
Rename this method? We're not getting a "helper"? We're getting a class name? (i think?)

Reply:
I don't want to rename stuff that doesn't need to be renamed in an already very complex PR. Maybe raise a card if you feel strongly about the name of this method?

Reply:
ok leave as is